server/routes/private.js

```diff
@@ -110,3 +110,52 @@ router.get('/test', async (req, res) => {
 
 module.exports = router
 
+router.post('/api/add-recurring-schedule', async (req, res) => {
+    // Validate appropriate parameters are passed to view workers at each school 
+    const paramSchema = Joi.object({
+        schoolId: Joi.number().integer().required(), //The student must specify their school ID in order to view the workers
+        schoolName: Joi.string().max(30), // The default value is available which is handled by the database Create Table logic 
+    });
+
+    const query = req.query ? req.query : {};
+
+    const schoolId = query.schoolId ? query.schoolId : null;
+    const schoolName = query.schoolName ? query.schoolName : null;
+
+    const { error } = paramSchema.validate({schoolId, schoolName});
+
+    if (!_.isNil(error)) res.send(error);
+
+    // Attempts to insert the available workers at each school 
+    const isSuccessfullyInserted = await schoolHandler.inputSchool(schoolId,schoolName);
+
+    res.send(isSuccessfullyInserted);
+});
+
+router.post('/api/worker-availability', async (req, res) => {
```

Comment:
Hi Simran,
Slightly confused about why this method was also added? Make sure to pull from master and update your branch to have the latest changes.

Reply:
Thanks Amy! I think I have merged the master branch. Should it be saying '/worker-availability'?

Reply:
Hi Simran,
It seems like the merge conflicts were not properly resolved. The same methods are listed above before the module gets exported. I would recommend using the Files Changed feature on GitHub to determine what code changes you have made. If anything seems out of the ordinary, then make sure to resolve them/remove these files as it may be due to undesired side effects of resolving merge conflicts.

Reply:
Thanks Amy for your comment! I will look into this! :)